$NetBSD: patch-as,v 1.1 2008/08/17 04:01:37 dholland Exp $

This may or may not work, but appears to be the best possible
approach; the code is doing gross marshalling/closure stuff.

--- rf/uevent.cpp~	2001-09-19 01:13:09.000000000 -0400
+++ rf/uevent.cpp	2008-08-16 23:50:15.000000000 -0400
@@ -375,7 +375,7 @@ dispatch:
 		(this->*mmf.pfnV_ii)(ULOWORD(arg1), UHIWORD(arg1));
 		break;
 	case epV_iii:
-		(this->*mmf.pfnV_iii)(ULOWORD(arg1), UHIWORD(arg1), (int)arg2);
+		(this->*mmf.pfnV_iii)(ULOWORD(arg1), UHIWORD(arg1), (int)(intptr_t)arg2);
 		break;
 	case epV_iiVP:
 		(this->*mmf.pfnV_iiVP)(ULOWORD(arg1), UHIWORD(arg1), arg2);
